### PR TITLE
Add pcodedmp

### PR DESCRIPTION
**Pull request by jan-janssen**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by jan-janssen** [comment]
@conda-forge/staged-recipes, ready for review

File: recipes/pcodedmp/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pcodedmp-{{ version }}.tar.gz
+  sha256: 025f8c809a126f45a082ffa820893e6a8d990d9d7ddb68694b5a9f0a6dbcd955
+
+build:
```
[the remark this answers is not in the document]
Comment (xhochy) [reply recipes/pcodedmp/meta.yaml:12]: You need to add the entry points here.

[the remark this answers is not in the document]
Comment (jan-janssen) [reply recipes/pcodedmp/meta.yaml:12]: Thanks for the recommendation

**Comment by jan-janssen** [comment]
`pcodedmp 1.2.6 requires win-unicode-console, which is not installed.`

**Comment by jan-janssen** [comment]
But when I try to add `win-unicode-console` to the windows build I get the following error:
```
Traceback (most recent call last):
  File "C:\Miniconda\lib\site-packages\boa\cli\mambabuild.py", line 123, in mamba_get_install_actions
    solution = solver.solve_for_action(_specs, prefix)
  File "C:\Miniconda\lib\site-packages\boa\core\solver.py", line 234, in solve_for_action
    t = self.solve(specs)
  File "C:\Miniconda\lib\site-packages\boa\core\solver.py", line 216, in solve
    raise RuntimeError("Solver could not find solution." + error_string)
RuntimeError: Solver could not find solution.Mamba failed to solve:
 - pip
 - easygui
 - oletools >=0.54
 - colorclass
 - msoffcrypto-tool
 - python_abi 3.9.* *_cp39
 - win-unicode-console
 - pyparsing
 - olefile
 - python >=3.9,<3.10.0a0

with channels:
 - local
 - conda-forge
 - conda-forge

The reported errors are:
- Encountered problems while solving:
-   - nothing provides vc 9.* needed by python-2.7.12-0
- 
```

File: recipes/pcodedmp/meta.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (jan-janssen) [reply recipes/pcodedmp/meta.yaml:12]: The reason why it can not be `noarch` is that the `win_unicode_console` package is only available on windows.

**Comment by jan-janssen** [comment]
@xhochy I fixed the windows build - can you take another look?

**Pull request merged**